specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-07-01/computeRPCommon.json

```diff
@@ -1395,8 +1395,9 @@
           "description": "Specifies the Windows Remote Management listeners. This enables remote Windows PowerShell."
         },
         "enableVMAgentPlatformUpdates": {
+          "readOnly": true,
```

Comment:
>"readOnly": true [](http://example.com/codeflow?start=10&length=16)

Is this API version not yet shipped ? 

Cannot see how this is being used as there no other callers/users of this property are changed in this PR.

This might result in a breaking change and needs approval from breaking change team.